src/Libraries/Microsoft.Extensions.Http.Telemetry/Logging/Internal/Log.cs

```diff
@@ -40,22 +61,22 @@ public static void OutgoingRequestError(ILogger logger, LogRecord record, Except
 
             collector.AddRequestHeaders(record.RequestHeaders);
             collector.AddResponseHeaders(record.ResponseHeaders);
-            collector.Add(HttpClientLoggingDimensions.Host, record.Host);
             collector.Add(HttpClientLoggingDimensions.Method, record.Method);
+            collector.Add(HttpClientLoggingDimensions.Host, record.Host);
             collector.Add(HttpClientLoggingDimensions.Path, record.Path);
```

Comment:
I'm surprised we don't do any redaction in this code. Can't the path contain sensitive info for example?